.storybook/webpack.config.js

```diff
@@ -39,8 +51,14 @@ module.exports = ({config, mode}) => {
     include: [modulesPath, utilsPath],
     loader: require.resolve('babel-loader'),
     options: {
-      presets: [['react-app', {flow: false, typescript: true}]],
+      presets: [
+        '@babel/preset-typescript',
```

Comment:
Changed up the loader here, I'm not really sure of what all `react-app` does behind-the-scenes so I'm manually adding the presets/plugins needed for TS here.

Reply:
I'm for decreasing the magic. Magic is useful for getting started, but as things get more complex, it makes sense to decrease magic.

Ideally our transforms are pretty simple/standard.